ckanext/canada/helpers.py

```diff
@@ -243,6 +254,7 @@ def normalize_strip_accents(s: Union[str, bytes]) -> str:
     """
     if isinstance(s, str):
         return s
```

Comment:
this would skip the normalization. that doesn't seem right.

Reply:
Yeah, I probably should have searched for the usage of this helper anyways. Some of this stuff is pretty old code or even fully unused code at this point. Will check this out more

Reply:
Okay yeah this is used in the facets list template:
```
{%- macro _text_key(d) -%}
    {{- h.normalize_strip_accents(d.label) -}}
{%- endmacro -%}
{%- do items.sort(key=_text_key) -%}
```

which makes sense. So yeah it should for sure be something more like:
```
if isinstance(s, bytes):
    s = s.decode('utf-8')
```
and then continue with the normalization. This also is only strictly used in the sort method, so not like it is actually altering any values here.